provider/marathon/marathon.go

```diff
@@ -32,6 +33,10 @@ const (
 
 var _ provider.Provider = (*Provider)(nil)
 
+// Regexp used to extract the name of the service and the name of the property for this service
+// All properties are under the format traefik.<servicename>.frontent.*= except the port/portIndex/weight/protocol directly after traefik.<servicename>.
+var servicesPropertiesRegexp = regexp.MustCompile(`^traefik\.(?P<service_name>.*?)\.(?P<property_name>port|portIndex|weight|protocol|frontend\.(.*))$`)
```

Comment:
Updated with added test label to ensure that `test..port` does not result with a service label named `` (empty string)